### ISSUE TITLE
Incorrect RKI FAQ example for vaccination immunity number of days

### ISSUE THREAD

**Issue by MikeMcC399** [corona-warn-app/cwa-documentation#651]
## Where to find the issue

https://digitaler-impfnachweis-app.de/faq/

![image](https://user-images.githubusercontent.com/66998419/123664264-786ad400-d837-11eb-8fd6-384f33d2312d.png)

## Describe the issue

The FAQ from RKI's https://digitaler-impfnachweis-app.de/faq/ does not agree with the way that CovPass App works, nor with the way that CWA has been corrected in 2.4.x to work (see https://github.com/corona-warn-app/cwa-documentation/issues/638)!

The issue was pointed out by @mawis in https://github.com/corona-warn-app/cwa-app-android/issues/3561#issuecomment-869408236. 

In order to ensure that a minimum of 14 days have passed since vaccination, the day of vaccination cannot be counted, because we have to allow for the theoretical case that the vaccination was done at 23:59 on the vaccination day. So in the example of vaccination on June 1, the 14 days must be counted from June 2 to June 15 inclusive, and only on June 16 can full vaccination status be declared. Only at the start of the day 00:00 local time on June 16 can we be sure that a full 14 days have passed since vaccination was done.

## Suggested change

I will follow up with a mail to info@covpass-app.de as suggested on the website https://digitaler-impfnachweis-app.de/faq/.

Please follow up with a request from the Corona-Warn-App Open Source Team to RKI or to the CovApp Team to correct this FAQ entry so it is consistent with the apps, since it is such an important and basic topic and parameter.

**Comment by MikeMcC399** [comment]
The problem report has been acknowledged by e-mail reply from:
[CoronaWarnApp@rki.de](mailto:CoronaWarnApp@rki.de)
CoronaWarnApp
Informationsstelle
Robert Koch-Institut | Nordufer 20 | 13353 Berlin

**Comment by Ein-Tim** [comment]
@MikeMcC399 

> The problem report has been acknowledged by e-mail reply from:
CoronaWarnApp@rki.de

So you wrote an E-Mail to info@covpass-app.de and  CoronaWarnApp@rki.de responded? 
Interesting 😄

**Comment by MikeMcC399** [comment]
@Ein-Tim 
> So you wrote an E-Mail to info@covpass-app.de and CoronaWarnApp@rki.de responded?
Interesting 😄

CoronaWarnApp@rki.de was in CC: of the e-mail together with corona-warn-app.opensource@sap.com also in CC:
info@covpass-app.de was in the To: field of the e-mail.

**Comment by dsarkar** [comment]
@MikeMcC399 I can confirm that your email was received at corona-warn-app.opensource@sap.com and the issue was forwarded. Thanks.

**Comment by MikeMcC399** [comment]
@dsarkar 
Thank you for getting involved!

**Comment by MikeMcC399** [comment]
I didn't get any further feedback so I checked the RKI https://digitaler-impfnachweis-app.de/faq/ site again.

The FAQ article has been corrected and the description now agrees with the way that CWA 2.4.x works. It says:

"Wenn Sie zum Beispiel die letzte notwendige Corona-Impfung am 01. Juni erhalten haben, dann ist der vollständige Corona-Impfschutz erst am 16. Juni erreicht."

This is correct because neither the yellow vaccination booklet, nor the digital vaccination certificate record the time of vaccination, so to ensure that at least 14 days have elapsed since vaccination, the day of vaccination cannot be counted. The 14 days start to be counted on the day after vaccination and end on the 14th day after vaccination. Full vaccination is reached at the beginning of the 15th day after vaccination. 

In the example, the 15th day after vaccination on the 1st June is the 16th June.

![image](https://user-images.githubusercontent.com/66998419/124649056-4a6d3b80-de98-11eb-865a-40ffed2f7937.png)

Closing this issue as fixed now.
